generic_server.cc

```diff
@@ -21,23 +21,21 @@ connection::connection(server& server, connected_socket&& fd)
     , _fd{std::move(fd)}
     , _read_buf(_fd.input())
```

Comment:
> Closes <https://github.com/scylladb/scylladb/issues/10305>.

Please don't invent new syntax, our bots aren't intelligent enough to follow it.

Reply:
apologies; fixed in v3.

Reply:
@lersek, sorry for the nitpicking but we use `Fixes #10305` rather than `Closes ...` when it is a bug fix.
`Closes` is used when a patch closes a PR, or when we want to close an issue without fixing it (akin to closing as "Not Planned").

Reply:
will update iv 

> we use `Fixes #10305` rather than `Closes ...` when it is a bug fix.
> `Closes` is used when a patch closes a PR, or when we want to close an issue without fixing it

will update in v4

Reply:
done in v4